src/gsNurbs/gsKnotVector.h

```diff
@@ -702,17 +702,17 @@ class gsKnotVector : public gsDomain<T>
     }
 
     /// Because of type compatibility, cf. the other version.
-    void refineSpans( const std::vector<unsigned> & spanIndices, mult_t knotsPerSpan = 1)
+    /*void refineSpans( const std::vector<index_t> & spanIndices, mult_t knotsPerSpan = 1)
```

Comment:
Why is this commented out ?

Reply:
Because of https://github.com/gismo/gismo/blob/a5b765bc2809f181ab008863b78146394c7ebdfd/src/gsNurbs/gsKnotVector.h#L719, it's the same. Ok, I may let it be `unsigned`